RELEASE.md

```diff
@@ -1,6 +1,57 @@
 # Release 1.6.0
 
-The 1.6 minor series tracks TensorFlow 1.6. Release notes upcoming.
+NOTICE: TensorBoard 1.6.0+ has moved to the `tensorboard` package name on PyPI:
+https://pypi.python.org/pypi/tensorboard. Only bugfix updates on 1.5.x will be
+applied to the old package name (`tensorflow-tensorboard`). To upgrade to
```

Comment:
Optional alternative to consider:

```markdown
...
applied to the old package name (`tensorflow-tensorboard`). To upgrade:

    pip uninstall tensorflow-tensorboard
    pip install tensorboard

See also "Known Issues" below, if you run into problems.
```

Reply:
Ack.  I already used the inline style for 1.5.1 relnotes, which were approved twice already, and the PyPI description for 1.5.1, and I think it's better to be consistent.